### PR TITLE
Revised the german lang file

### PR DESCRIPTION
**Pull request by X00LA**
Corrected a lot of the poor Google-Translator grammar.
Corrected spelling errors.
Translated forgotten words.

### CODE REVIEW

File: de_DE.yml [hunk 1]
```diff
@@ -1,769 +1,727 @@
...
-verbose-upload-start: "&bVerboselog &cdeaktiviert&b. Lade Ergebnisse hoch..."
-verbose-results-url: "&aURL der Verboseergebnisse:"
-tree-upload-start: "&bErstelle Berechtigungsbaum..."
-tree-empty: "&cBerechtigungsbaum konnte nicht erstellt werden. Keine Ergebnisse gefunden."
-tree-url: "&aURL des Berechtigungsbaumes:"
-search-searching: "&aSuche nach Spielern und Gruppen mit der &bBerechtigung {0}&a..."
-search-searching-members: "&aSuche nach Spielern und Gruppen, die von &b{0}&a erben..."
-search-result: "&aEs wurden &b{0}&a Einträge von &b{1}&a Spielern und &b{2}&a Gruppen gefunden."
-search-showing-users: "&bZeige Spielereinträge:    &7(Seite &f{0}&7 von &f{1}&7 - &f{2}&7 Einträge)"
-search-showing-groups: "&bZeige Gruppeneinträge:    &7(Seite &f{0}&7 von &f{1}&7 - &f{2}&7 Einträge)"
```
Comment (Andre601) [inline de_DE.yml]: Why did you remove those?

**Comment by Andre601** [comment]
The changes you made are bad.

To list a few issues:
- You removed a huge chunk of stuff wich, compared to the original [en_US.yml](https://github.com/LuckPerms/locale/blob/master/en_US.yml) is needed.
- The changes you made are non-existant, compared to the current [de_DE.yml](https://github.com/LuckPerms/locale/blob/master/de_DE.yml)
- Was your change **really** just to sort the stuff by alphabet?

I personally recommend denying this PR since it - as far as I can see - just changes the order of the text, which isn't really something important.

File: de_DE.yml [hunk 2]
```diff
@@ -1,769 +1,727 @@
...
+delete-success: "&b{0}&a wurde erfolgreich gelöscht."
+does-not-exist: "&4{0}&c existiert nicht!"
+does-not-have-chat-meta: "&b{0}&a hat {1} &f'{2}&f'&a nicht mit einer Priorität von &b{3}&a, im Kontext {4}&a gesetzt."
+does-not-have-permission: "&b{0}&a hat &b{1}&a, im Kontext {2}&a, nicht gesetzt."
+does-not-have-temp-chat-meta: "&b{0}&a hat {1} &f'{2}&f'&a nicht temporär mit einer Priorität von &b{3}&a, im Kontext {4}&a gesetzt."
+does-not-have-temp-permission: "&b{0}&a hat &b{1}&a, im Kontext {2}&a, nicht temporär gesetzt."
+does-not-inherit: "&b{0}&a erbt nicht, im Kontext {2} von &b{1}&a."
+does-not-temp-inherit: "&b{0}&a erbt, im Kontext {2} von &b{1}&a, nicht temporär."
+doesnt-have-meta: "&b{0}&a hat den Metaschlüssel &f'{1}&f'&a im Kontext {2}&a nicht gesetzt."
+doesnt-have-temp-meta: "&b{0}&a hat den Metaschlüssel &f'{1}&f'&a im Kontext {2}&
```
Comment (Andre601) [inline de_DE.yml]: Sereously? Just moving the text to here?

**Comment by X00LA** [comment]
I haven't removed anything from the file.
The alphabetical order was inserted by my yaml checker. :/ Forgot to uncheck the option.
Hope now it will pass the check.
Compared it with the newest en_EN.yml.

**Comment by lucko** [comment]
I've fixed the merge conflicts, @Andre601 does this look good now?

**Comment by Andre601** [comment]
Looks ok imo.
But perhaps BrainStone should also check thst.
